app/assets/stylesheets/admin/_variables.scss

```diff
@@ -3,3 +3,9 @@
 $content-wrapper-padding: 20px;
 $sidebar-width: 190px;
 $app-fonts: 'Open Sans', sans-serif;
+
+// Common styles
+.draft {
```

Comment:
Intended to dry up step draft styles. We use something similar in both the timeline in the calendar. If we change either in the future, the change can touch both places at once.

Reply:
Great idea! If you intend for this style to just be applicable to project steps, It should maybe go in the project steps folder? Or, if it's going to be an app wide style for draft-type things, Then maybe it should go in _general.scss?